src/components/CellLink/CellLink.module.less

```diff
@@ -21,7 +21,6 @@
 	margin: 3em 0;
 
```

Comment:
Is there a reason you didn't delete the css related to Linkbox?
I think you can just leave the .cell {} block.

Reply:
No, there's no reason.
I left only the cell class.